### PR TITLE
Set up off Vagrant and inital Ansible script.

### PR DESCRIPTION
**Pull request by janvandermeulen**
- Vagrant now sets up the controller and workers.
- Setup of Ansible scripts which can connect to the controller and workers. 
- The Ansible scripts are not yet done, but as this is not my responsibility it will be for @sr1998 to finish this part.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
```
Comment (sr1998) [inline Vagrantfile]: Should the "2" be a variable?

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      vb.cpus = 1
+    end
+    controller.vm.provision "ansible" do |ansible|
+      ansible.playbook = "ansible/playbook-controller.yml"
+      ansible.inventory_path = "ansible/inventory.cfg"
+    end
+  end
+
+  # Define workers
+  (1..2).each do |i|
```
Comment (sr1998) [inline Vagrantfile]: According to the assignment, we should have this 2 as a variable:
"The number of provisioned workers should be configurable through a variable in the Vagrantfile to make it easy to scale (up or down)" (page 1)

File: Vagrantfile (hunk 1 above)
Comment (janvandermeulen) [reply Vagrantfile]: Probably unnecessary, but fixed anyways.

File: Vagrantfile (hunk 2 above)
Comment (janvandermeulen) [reply Vagrantfile]: Fixed in commit #[4ce43e9](https://github.com/remla24-team10/operation/pull/3/commits/4ce43e934db90b9c82bdf92c601797b1232e3e5e) with a python script that gets called from the Vagrantfile.

**Review by sr1998 — APPROVED** [review]

**Pull request merged**